Jasmine/Common/Extensions/Array+Extensions.swift

```diff
@@ -0,0 +1,25 @@
+func ==<T: Equatable>(lhs: [T], rhs: [T?]) -> Bool {
+    guard lhs.count == rhs.count else {
+        return false
+    }
+
+    for idx in 0..<lhs.count {
+        guard let rhsElement = rhs[idx], lhs[idx] == rhsElement else {
+            return false
+        }
+    }
+
+    return true
+}
+
+func ==<T: Equatable>(lhs: [T?], rhs: [T]) -> Bool {
```

Comment:
yeah would need another func for that, but we shouldn't add it until we actually need to compare 2 array of optionals?